src/modm/platform/extint/rp/module.md

```diff
@@ -0,0 +1,54 @@
+# External Interrupt Handler
+
+This driver provides an API for configuring all IRQ lines via register access.
+Note that you need to pass a mask, which allows you to configure multiple IRQ
+lines at once.
```

Comment:
Copy-paste from STM32? 🤪
This code does not allow configuring multiple handlers at the same time?